src/main/java/nextstep/subway/member/client/github/GithubClient.java

```diff
@@ -0,0 +1,66 @@
+package nextstep.subway.member.client.github;
+
+import nextstep.subway.member.client.github.config.GithubClientProperties;
+import nextstep.subway.member.client.github.dto.GithubAccessTokenRequest;
+import nextstep.subway.member.client.github.dto.GithubAccessTokenResponse;
+import nextstep.subway.member.client.github.dto.GithubProfileResponse;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.stereotype.Component;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Optional;
+
+@Component
+public class GithubClient {
+
+    public static final String ACCESS_TOKEN = "/github/login/oauth/access_token";
+    public static final String FIND_USER = "/github/user";
```

Comment:
```suggestion
    public static final String ACCESS_TOKEN_PATH = "/github/login/oauth/access_token";
    public static final String FIND_USER_PATH = "/github/user";
```
처럼 좀 더 자세한 변수명을 지으면 더 한눈에 들어올 것 같아요 :)